docs/interceptors.md

```diff
@@ -0,0 +1,511 @@
+<!--
+---
+linkTitle: "Configuring Interceptors"
+weight: 5
+---
+-->
+# Configuring `Interceptors`
+
+- [Overview](#overview)
+- [Specifying an `Interceptor`](#specifying-an-interceptor)
+- [Webhook `Interceptors`](#webhook-interceptors)
+- [GitHub `Interceptors`](#github-interceptors)
+- [GitLab `Interceptors`](#gitlab-interceptors)
+- [Bitbucket `Interceptors`](#bitbucket-interceptors)
+- [CEL `Interceptors`](#cel-interceptors)
+- [Implementing custom `Interceptors`](#implementing-custom-interceptors)
+
+## Overview
+
+An `Interceptor` is a "catch-all" event processor for a specific platform that runs before the `TriggerBinding.` It allows you to perform payload filtering, verification (using a secret),
+transformation, define and test trigger conditions, and implement other useful processing. Once the event data passes through an `Interceptor`, it then goes to the `Trigger` before you pass
+the payload data to the `TriggerBinding`. You can also use an `Interceptor` to modify the behavior of the associated `Trigger`.
+
+Tekton Triggers currently supports two distinct `Interceptor` implementations:
+- Standalone `Interceptors`, which you define as separate Kubernetes objects and then reference, along with the desired parameters, within your `EventListener`. See [TEP-0026](https://github.com/tektoncd/community/blob/main/teps/0026-interceptor-plugins.md) for more information.
```

Comment:
Instead of standalone interceptors, can we call mention these are basically ClusterInterceptors?

Reply:
Rewrote this section to make this more up-front.